src/updates-via.js

```diff
@@ -1,9 +1,9 @@
 /*
  * Updates-Via
  */
-const namedNode = require('./data-factory').namedNode
+import { namedNode } from './data-factory'
```

Comment:
There's a bug here, reported here: https://gitter.im/linkeddata/rdflib.js?at=5d5c1e77950718247717e691

`namedNode` is not a named export of `data-factory` - instead, it offers a default export of an object with a property `namedNode`.

Reply:
True; the export is correct, the import is not. On it.

Reply:
Fix in e7c5680122def0a2b1298fb5c7158203dd24c707.